src/target/ext/register_info_override.rs

```diff
@@ -0,0 +1,59 @@
+//! (LLDB extension) Override the register info specified by `Target::Arch`.
+use crate::arch::lldb::Register;
+use crate::target::Target;
+
+/// (LLDB extension) If your target is using the `qRegisterInfo` packet, this
```

Comment:
nit: similarly, since the module these types are contained within is marked as `(LLDB extension)`, there's no need to repeat that comment within the module.